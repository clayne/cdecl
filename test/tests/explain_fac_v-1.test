cdecl @ @ @ explain void f(int x[const]) @ 0
