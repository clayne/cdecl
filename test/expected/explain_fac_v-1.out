declare f as function (x as array constant of integer) returning void
